rfcs/text/formik.md

```diff
@@ -0,0 +1,695 @@
+- Start Date: 2019-11-15
+
+
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Detailed design](#detailed-design)
+  - [Overview of Carbon's form and validation functionality](#overview-of-carbons-form-and-validation-functionality)
+  - [Overview of Formik](#overview-of-formik)
+    - [Form validation using Formik](#form-validation-using-formik)
+  - [Code examples](#code-examples)
+    - [Checkbox and CheckboxGroup](#checkbox-and-checkboxgroup)
+      - [HTML and React](#html-and-react)
+      - [Checkbox usage with Formik](#checkbox-usage-with-formik)
+      - [Form-level validation examples](#form-level-validation-examples)
+      - [Simplification using `<Field>`](#simplification-using-field)
+      - [Field-level validation example](#field-level-validation-example)
+      - [CheckboxGroup example: uncontrolled](#checkboxgroup-example-uncontrolled)
+      - [CheckboxGroup example: controlled](#checkboxgroup-example-controlled)
+    - [Blocking navigation on unsaved changes](#blocking-navigation-on-unsaved-changes)
+- [Drawbacks](#drawbacks)
+  - [No support for non-blocking ("warning" or "info") validations](#no-support-for-non-blocking-%22warning%22-or-%22info%22-validations)
```

Comment:
I think the entity characters are screwing with the skip-links here

Reply:
@ianforster-sage  not sure what you mean, the links seem to navigate to the correct sections and the text seems fine, could you clarify for me as I may be missing something?